certif/app/controllers/authenticated/sessions/finalize.js

```diff
@@ -18,6 +18,8 @@ export default class SessionsFinalizeController extends Controller {
   examinerGlobalCommentMaxLength = 500;
   issueReportDescriptionMaxLength = 500;
   @tracked showConfirmModal = false;
+  @tracked hasIncident = false;
+  @tracked hasJoiningIssue = false;
```

Comment:
(api: add new booleans to session finalize controller) Fail : remettre cette modiflà d'où elle vient